apps/site/assets/ts/schedule/components/line-diagram/StopPredictions.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Headsign, Route } from "../../../__v3api";
+import {
+  timeForCommuterRail,
+  statusForCommuterRail
+} from "../../../helpers/prediction-helpers";
+
+interface Props {
+  headsigns: Headsign[];
+  route: Route | null;
```

Comment:
Since this is only used to determine whether the route is a commuter rail route or not, would it make more sense to have a boolean prop like `isCommuterRail` here instead?

Reply:
Maybe. The `if (route.type === 2)` logic does have to live somewhere — do you think it should be at the point where the `StopPredictions` component is invoked?

Reply:
Yes.